.changeset/purple-coins-change.md

```diff
@@ -0,0 +1,6 @@
+---
+"@utrecht/storybook-html": minor
+"@utrecht/paragraph-css": minor
+---
+
+removed lead and small from paragraph component and story
```

Comment:
Deze changelog is nog niet compleet, en `minor` klopt niet. Ik zal kijken of ik donderdag betere documentatie kan schrijven met een stappenplan dat je kan volgen.

Reply:
Ik heb in #2309 die documentatie toegevoegd, ik ben benieuwd of je daarmee een plan kan maken voor deze PR!

Reply:
@scar055 Mooie verbetering. Heb je ook het stukje gelezen over "Impact op dependencies"?